packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -948,16 +948,6 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return this._storage!;
     }
 
-    public get reSubmitFn(): (
```

Comment:
Have we warned partners this API is going away?  I don't see it marked as deprecated.

Reply:
I put it back. We can consider it a separate issue to get rid of it. We're no longer using it in our codebase. But I don't know what to tell partners to use instead if they need this.